nextstrain_profiles/preprocess/rules.yaml

```diff
@@ -0,0 +1,5 @@
+# Note that `custom_rules` from multiple configuration YAMLs are combined by replacement,
+# not by concatenation
+custom_rules:
+  - nextstrain_profiles/nextstrain/download_preprocessed.smk
```

Comment:
@jameshadfield I'm sorry if I missed a step in the conversation above, but does the preprocess profile need to have the download rules if the goal of this profile is to re-run analyses without the download?

Reply:
Yeah it does, but it's a little convoluted why this is. Without it (and therefore the `download_<STEP>` rules aren't available) then the ruleorder blocks which refer to them in `preprocess/rules.smk` and `nextstrain-scicore/localrules.smk` cause problems. The former is easy to remove, but the latter file is coming via a `localrules` config entry in a configfile imported by `preprocess/config.yaml`. So we could switch that latter import to be via `custom_rules`, but then I think we'd need to add these rules to it as well:

```
  - workflow/snakemake_rules/export_for_nextstrain.smk
  - nextstrain_profiles/nextstrain/subsampling_ranges.smk
```

Reply:
We might be able to simply switch `custom_rules` to `localrules` (in `nextstrain_profiles/preprocess/rules.yaml`), which would be cleaner.